### ISSUE TITLE
Add #acp

### ISSUE THREAD

**Issue by Greenscreen23** [Grisely/packages#13]
Hi, thank you for the great package :)

#7 suggested adding some commands from tex. I find those very nice, and use them most of the time, but I find myself missing `#acp` (just do the acronym in plural and decide for yourself whether its short or full).

I also noticed that `#acl` and #aclp` (long form without the acronym after is also missing. I don't necessarily need them right now, but they might also be a nice addition. 

Let me know if you want me to open a PR or if you want to do this. I am quite new to typst, but I'll probably figure it out :)

**Comment by Grisely** [comment]
Indeed, `#acp` is missing. It's easy enough to add it to the next version.
